### DISCUSSION TITLE
@types/mongodb older version

### DISCUSSION THREAD

**Discussion by kobi-co** [DefinitelyTyped/DefinitelyTyped#64241] [Issues with a @types package]
Hi, 
After the latest release of mongo db 5.0.0 
It seems like the @types/mongodb has some issues. 
The issue is that in [version 3.6.20](https://www.npmjs.com/package/@types/mongodb/v/3.6.20?activeTab=explore) of @types/mongodb, it has a dependency for `"@types/bson": "*"`
The latest `@types/bson` has removed ObjectID and it causes issues with `@types/mongodb`.

Is there any chance to patch `@types/mongodb` to have a dependency to `@types/bson:4.0.5` or something like that ?

Thanks for the support

[a comment by an automated account is not shown]

**Comment by mmelikyan** [comment]
For future visitors - I was able to overcome this issue by pinning a specific version of BSON to `package.json`. 

if using NPM add to `package.json`
```json
"overrides": {
  "@types/bson": "4.0.5"
}
```

if using PNPM add to `package.json`
```json
"pnpm": {
    "overrides": {
      "@types/bson": "4.0.5"
    }
}
```
